src/rust/collections/async_queue.rs

```diff
@@ -39,7 +39,6 @@ pub struct AsyncQueue<T> {
     waiters: Vec<YielderHandle>,
 }
 
-#[derive(Clone)]
```

Comment:
Why can't we just `derive`?

Reply:
Oh, it seemed to require that T be clonable as well. Not sure why